src/main/java/org/broadinstitute/hellbender/tools/spark/sv/ReadClassifier.java

```diff
@@ -125,25 +125,33 @@ private void checkBigIndel( final List<CigarElement> cigarElements,
     private void checkDiscordantPair( final GATKRead read, final List<BreakpointEvidence> evidenceList ) {
         if ( read.mateIsUnmapped() ) {
             evidenceList.add(new BreakpointEvidence.MateUnmapped(read, readMetadata));
-        } else if ( isInterContig(read.getContig(),read.getMateContig()) ) {
-            evidenceList.add(new BreakpointEvidence.InterContigPair(read, readMetadata));
-        } else if ( read.isReverseStrand() == read.mateIsReverseStrand() ) {
-            evidenceList.add(new BreakpointEvidence.SameStrandPair(read, readMetadata));
-        } else if ( read.isReverseStrand() ?
-                read.getStart() + allowedShortFragmentOverhang < read.getMateStart() :
-                read.getStart() - allowedShortFragmentOverhang > read.getMateStart() ) {
-            evidenceList.add(new BreakpointEvidence.OutiesPair(read, readMetadata));
         } else {
-            final float zIshScore = readMetadata.getZishScore(read.getReadGroup(), Math.abs(read.getFragmentLength()));
-            if ( zIshScore > MAX_ZISH_SCORE && zIshScore < MIN_CRAZY_ZISH_SCORE ) {
-                evidenceList.add(new BreakpointEvidence.WeirdTemplateSize(read, readMetadata));
+
+            final int contigID1 = readMetadata.getContigID(read.getContig());
+            final int contigID2 = readMetadata.getContigID(read.getMateContig());
+
+            if (contigID1 != contigID2) {
+                if (isValidIntercontigEvidence(contigID1, contigID2)) {
```

Comment:
I'd suggest killing the tiny function, isValidIntercontigEvidence, which now contains a superfluous condition that's already been tested (equality of contigs) and inline this code:
```
    if ( !readMetadata.ignoreCrossContigID(contigID1) && !readMetadata.ignoreCrossContigID(contigID2) )
```
I think this would read more clearly, but YMMV.  I don't insist.  And by the way, thanks for finding and fixing my faulty logic.